ml-agents/mlagents/trainers/torch/utils.py

```diff
@@ -56,8 +56,9 @@ def create_encoders(
         h_size: int,
         num_layers: int,
         vis_encode_type: EncoderType,
-        action_size: int = 0,
-    ) -> Tuple[nn.ModuleList, nn.ModuleList, nn.ModuleList]:
+        encoded_act_size: int = 0,
+        normalize: bool = False,
+    ) -> Tuple[nn.ModuleList, nn.ModuleList]:
```

Comment:
Modify the docstring here, the inputs and outputs of this method changed sllightly

Reply:
Updated